Tests/scripts/server_get_artifact.sh

```diff
@@ -3,11 +3,12 @@ set -e
 
 ACCEPT_TYPE="Accept: application/json"
 SERVER_API_URI="https://circleci.com/api/v1/project/demisto/server"
-TOKEN_ATTR="circle-token=$SERVER_CI_TOKEN"
```

Comment:
out of curiosity, no access to circle vars inside script?

Reply:
exactly